### DISCUSSION TITLE
How do I change the audio language?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5722] [General]
I've got a video with Eng and Hindi audio, but it's defaulting to Hindi and I need Eng. How do I switch it? mpv or vlc

**Comment by darnn** [comment]
You'll have to switch to MPV as your video player (in options->settings->video player, but I suspect you know that already).
Then you'll be able to do this:
![se-change audio](https://user-images.githubusercontent.com/19672444/150229971-ccaeded0-6176-4098-ac2b-b1afd189f609.png)

**Comment by DrReddP** [comment]
Balls out, perfect. Thanks man
